### PR TITLE
hot-fix rendering by year

### PR DESCRIPTION
**Pull request by hegner**

### CODE REVIEW

File: _gsocproposals/2018/proposal_ATLASEigen.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title:  Faster matrix algebra for ATLAS
 layout: gsoc_proposal
 project: ATLAS
+year: 2018
```
Comment (graeme-a-stewart) [inline _gsocproposals/2018/proposal_ATLASEigen.md]: Do we need the year metadata anymore? It seems like you now get this during processing from the path.

Comment (graeme-a-stewart) [reply _gsocproposals/2018/proposal_ATLASEigen.md]: If we decide we don't need it then it should be removed from the 2017 proposals and the instructions.

**Comment by graeme-a-stewart** [comment]
There is a problem in the 2017/summary.md page - the skipped 2018 projects are causing a blank line in the markdown, which is then splitting the table into multiple pieces. I'll need to work on a fix for that - but we should decide if the project proposals should have year attributes in them first.

File: _gsocproposals/2018/proposal_ATLASEigen.md (hunk 1 above)
Comment (hegner) [reply _gsocproposals/2018/proposal_ATLASEigen.md]: If we fix the usage of page.year in _layouts/gsoc_proposal.html we should be able to drop the year from the metadata.

**Comment by hegner** [comment]
see https://shopify.github.io/liquid/basics/whitespace/

**Comment by graeme-a-stewart** [comment]
Yes, I just worked out the gymnastics. Seeing if I can do it without the `page.year` attribute...

**Comment by graeme-a-stewart** [comment]
Fix for summary is in https://github.com/hegner/hep-sf.github.io/pull/7

**Pull request merged**